Fix nginx conf for icon display in releases >=19.09

drosofff opened the pull request:
- I found these patches are enough to display correctly small icons in the `manage tool` panel.
Not sure it is the more elegant patch to adapt to the new path of these png files, but it does not hurt previous releases either

- The PR also fixes the ci_test_galaxykickstart.sh by using ansible 2.7.4 instead of ansible 2.9.2. A side info from this fix is that the ansible-galaxy-tools role should be fixed to accommodate ansible 2.9.2 - if desired (probably a problem of access rights differently managed by the two ansible versions).

drosofff committed:
add location in nginx.conf.j2 for correct loading of some /blue/ icons

drosofff committed:
remove 2 tests from the bioblend test suite

drosofff committed:
fix ci_test_galaxykickstart.sh

dannon commented:
Is this the same problem as addressed in https://github.com/galaxyproject/galaxy-helm/pull/87/files?

drosofff commented:
@dannon I cannot say...

drosofff committed:
test in GCE and adapt ci_test_galaxykickstart.sh to avoid wrong ansible dependencies

drosofff committed:
fix tool install by pinning to ansible 2.7.4

install ansible 2.7.4 earlier

reintegrate 2 bioblend tests

make a trial with all bioblend tests

select minimum set of bioblend tests to filter out

drosofff committed:
resolve conflict for ci_test_galaxykickstart.sh

drosofff committed:
pin bioblend for galaxykickstart test

drosofff committed:
fix incorrect pinning of bioblend==0.13.0

abretaud commented:
@drosofff could you rebase/resolve conflicts?
Tests are passing now on master, so I guess pining to an older ansible is no longer needed

drosofff commented:
Hi @abretaud I have a few hours to fix this. can you refresh my memory with several human readable sentences to summarize the situation: this pr is quite old and I confess that it is not in the center of my attention anymore...

abretaud commented:
Yes, looking closely at the diff, I don't think the changes are needed anymore:
- the tests are passing now
- there where some changes with the images path

So I'm closing this PR, feel free to open a new one if you find out that some changes are still needed
